errata_tool/product_version.py

```diff
@@ -26,6 +26,69 @@ def releasedBuilds(self):
         result = self._get(url)
         return result
 
+    def variants(self):
+        """Get the list of variants for this Product Version.
+
+        :returns: a (possibly-empty) list of Variant objects. Each Variant
+                  contains a dict, for example::
```

Comment:
Let's move this docstring to the `Variants` class definition, since it will be true for all `Variants`

Reply:
Like this: https://github.com/red-hat-storage/errata-tool/commit/0d3a6bcb8eb8df484301e2f0d65df5768668c4bc